Prac05/birth_date_dict.py

```diff
@@ -0,0 +1,24 @@
+"""
+Program asks user for the date-of-birth details for 5 people, and displays their individual ages by using a dictionary.
+"""
+
+#TODO Finish this!!!!
+
+birth_date_dict = {}
+
+
+def add_date(name, birth_date):
+    birth_date_dict[name] = birth_date
+    print(birth_date_dict)
+
+
+def main():
+    name = input("Enter the person's name: ")
+    birth_date = input("Enter the person's date of birth: ")
+    for i in range(5):
+        add_date(name, birth_date)
+        name = input("Enter the person's name: ")
+        birth_date = input("Enter the person's date of birth: ")
+        i += 1
```

Comment:
Why do you need the i += 1 ? when you call for i in range the i variable is going to be incremented by default after every run of the loop